### ISSUE TITLE
1.3.0.2011008-SNAPSHOT aar missing from sonatype

### ISSUE THREAD

**Issue by loicsiret** [eneim/kohii#153]
Hello,

since this morning I got a 
``` 
Could not find kohii-core-1.3.0.2011008-SNAPSHOT.jar (im.ene.kohii:kohii-core:1.3.0.2011008-SNAPSHOT:20210426.094036-15).
  Searched in the following locations:
      https://oss.sonatype.org/content/repositories/snapshots/im/ene/kohii/kohii-core/1.3.0.2011008-SNAPSHOT/kohii-core-1.3.0.2011008-20210426.094036-15.jar
```

Can you please re upload the snapshot ?

Best regards,

Loïc

**Comment by JSBerrocoso** [comment]
Hi!!
Good job!! Nice library!
Please re upload the snapshot, I have the same problem :(

   > Could not find kohii-core-1.3.0.2011008-SNAPSHOT.jar (im.ene.kohii:kohii-core:1.3.0.2011008-SNAPSHOT:20210426.094036-15).
     Searched in the following locations:
         https://oss.sonatype.org/content/repositories/snapshots/im/ene/kohii/kohii-core/1.3.0.2011008-SNAPSHOT/kohii-core-1.3.0.2011008-20210426.094036-15.jar

Possible solution:
 - Declare repository providing the artifact, see the documentation at https://docs.gradle.org/current/userguide/declaring_repositories.html


Thanks

Best regards

**Comment by eneim** [comment]
Hmm thanks for notifying , let me double check.

**Comment by eneim** [comment]
It would be fixed now: https://oss.sonatype.org/content/repositories/snapshots/im/ene/kohii/kohii-core/1.3.0.2011008-SNAPSHOT/

**Comment by xibaca** [comment]
Hello, I have a question. When snapshot artifacts are published for :kohii-core, are corresponding versions for :kohii-exoplayer and other modules also meant to be published as well? Because if that's the case, I think there's some problem preventing this from happening automatically.

I'm asking because I noticed im.ene.kohii:kohii-core:1.3.0.2014000-SNAPSHOT is available while the latest :kohii-exoplayer snapshot version available is im.ene.kohii:kohii-exoplayer:1.3.0.2011008-SNAPSHOT.

Trying to use this combination of kohii-core:1.3.0.2014000-SNAPSHOT and kohii-exoplayer:1.3.0.2011008-SNAPSHOT (plus exoplayer:2.14.0) is leading to runtime crashes which I imagine would not happen if I used latest kohii-exoplayer instead (1.3.0.2014000-SNAPSHOT).
